priv/repo/seeds.exs

```diff
@@ -1,11 +1,4 @@
-# Script for populating the database. You can run it as:
-#
-#     mix run priv/repo/seeds.exs
-#
-# Inside the script, you can read and write to any of your
-# repositories directly:
-#
-#     CoursePlanner.Repo.insert!(%CoursePlanner.SomeModel{})
-#
-# We recommend using the bang functions (`insert!`, `update!`
-# and so on) as they will fail if something goes wrong.
+CoursePlanner.Repo.delete_all CoursePlanner.User
```

Comment:
Sometimes I pollute the database with further tests, so I just run the seeds again instead of dropping and setting up the database again. I wouldn't necessarily say it's good practice, but mostly common practice.
But maybe we shouldn't have this seed at all, since it's run in every environment. Do you know any way to run separate seeds for dev/test/prod?